catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/common/NitfHeaderAttribute.java

```diff
@@ -393,19 +417,24 @@ private static String convertFormatVersion(final FileType format) {
     return "";
   }
 
-  private static Date convertNitfDate(DateTime nitfDateTime) {
-    if (nitfDateTime == null || nitfDateTime.getZonedDateTime() == null) {
-      return null;
+  private static Serializable handleReleasability(final String nitfReleaseInstructions) {
```

Comment:
❓ Should we add that method to the utility class and call it something like `toStandardReleasability`? If we do, we should add a JavaDoc comment and `throws NitfParsingException` to the signature.

Reply:
I would say no because the logic specifically deals with just the Release Instructions attribute specified in the NITF specification. I had not looked at other attributes to see if this method could be used, so I think it would be safer to keep and refactored out later if needed.